nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
```

Comment:
I think this functionality is broken. Not by this PR, but already in the main branch. Shall we leave it here and repair it in an additional PR afterwards?